### PR TITLE
Release v1.19.1

### PR DESCRIPTION
**Pull request by qrkourier**

### CODE REVIEW

File: netfoundry.platform/galaxy.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ namespace: netfoundry
 name: platform
 
 # The version of the collection. Must be compatible with semantic versioning
-version: 1.19.0
+version: 1.19.1
```
Comment (tmousawNF) [inline netfoundry.platform/galaxy.yml:11]: Bump the major version to indicate to the user that there may be a compatibility issue.

File: netfoundry.platform/plugins/modules/netfoundry_service_simple.py [hunk 2]
```diff
@@ -289,3 +289,4 @@ def main():
 
 if __name__ == '__main__':
     main()
+a
```
Comment (tmousawNF) [inline netfoundry.platform/plugins/modules/netfoundry_service_simple.py:292]: This seems like a stray "a".

**Review by tmousawNF — COMMENTED** [review]
Please resolve comments and let me know when you are done.

**Comment by qrkourier** [comment]
superseded by https://github.com/netfoundry/ansible-collection/pull/14

**Pull request closed without merging**